base64.c

```diff
@@ -24,7 +24,9 @@
  /**
  * 64 character table for converting to Base64
  */
-char b64_table[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+char b64_table[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZ\
```

Comment:
The way you've split the strings here will break the program, since that backslash only escapes the newline character, and not the rest of it. I'll give it a quick fix and it should be fine. 

The way to split strings up into C is to separate them into separate strings, like so:
```
char longString[] = "This is the first part of the string, but it's very long so "
        "I can split them up like so, and the compiler will realise that since there's "
        "nothing between them, they must all be part of a single string.";
```

As it is now, they all have a bunch of tab characters in them, which will throw off the base64 encoder.